File: prezi.core/model/presentation.go
package model

import (
	"time"
)

type Presentation struct {
	Id        string    `bson:"id" json:"id"`
	Title     string    `bson:"title" json:"title"`
	ThumbNail string    `bson:"thumbnail" json:"thumbNail"`
	CreatedAt time.Time `bson:"createdAt" json:"createdAt"`
	Creator   User      `bson:"creator" json:"creator"`
}

File: prezi.web/static/js/models/presentation.js
var app = app || {};

(function () {
	'use strict';

	app.Presentation = Backbone.Model.extend({
		defaults: {
			id: "",
			title: "Awesome presentation",
			thumbNail: "https://placeimg.com/400/400/any",
			createdAt: "Jan 1, 2016"
		}
	});
})();